apollo-federation/src/sources/connect/fetch_dependency_graph/mod.rs

```diff
@@ -34,12 +34,67 @@ impl FetchDependencyGraphApi for FetchDependencyGraph {
     fn can_reuse_node<'path_tree>(
```

Comment:
I don't think i like this method name — "can" implies a boolean return value. 

should it be "reusable_edges_for_node"?

what happens to the edges that can't be reused? what happens if we return an empty vec? 

cc @sachindshinde

Reply:
It originally returned a bool, though for connectors we don't batch across different aliases/arguments at the moment, so it ends up filtering edges instead.

I wouldn't necessarily call it `reusable_edge_for_node()`, as that would imply the edge is the thing being reused when it's really the node. It's more like those particular edges can reuse the node, so maybe `edges_that_can_reuse_node()`?

Reply:
yeah that's helpful!

Reply:
Done!